workspaces/lluis/Figure.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="typescript">
+  export let size
+  export let className
+  export let isRounded = false;
```

Comment:
Not sure if isRounded is really used anywhere. Please remove any code that's not really used. We don't want to maintain unused code :-)